cmd/juju/storage/volumelist.go

```diff
@@ -10,9 +10,16 @@ import (
 	"launchpad.net/gnuflag"
 
 	"github.com/juju/juju/apiserver/params"
+	"github.com/juju/juju/cmd/envcmd"
 )
 
-const VolumeListCommandDoc = `
+// VolumeListAPI defines the API methods that the volume list command use.
```

Comment:
I think this should probably be defined near the end of the file - where it was defined previously.